sample/01-monitoring/src/app.controller.ts

```diff
@@ -1,7 +1,15 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, UseInterceptors } from '@nestjs/common';
 import { AppService } from './app.service';
+import { ResponseTimeInterceptor } from '@samagra-x/stencil'; 
 
 @Controller()
+@UseInterceptors(
+  new ResponseTimeInterceptor(
+    'controller',
+    'http://localhost:7889',
```

Comment:
why is it hardcoded?

Reply:
I'm not able to use a env here

Reply:
Dependency Injection is not available in UseInterceptors method